<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(18, 130, 119)</string>
  </property>
  <widget class="QPushButton" name="add_to_breakfast">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>+ добавить еду</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_to_lunch">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>+ добавить еду</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_to_dinner">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>+ добавить еду</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_to_snack">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>+ добавить еду</string>
   </property>
  </widget>
  <widget class="QLabel" name="app_logo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poor Richard</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>MyDiet</string>
   </property>
  </widget>
  <widget class="QLabel" name="breakfast">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>Завтрак:         ккал</string>
   </property>
  </widget>
  <widget class="QLabel" name="lunch">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>Обед:           ккал</string>
   </property>
  </widget>
  <widget class="QLabel" name="dinner">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>Ужин:           ккал</string>
   </property>
  </widget>
  <widget class="QLabel" name="snack">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>Перекус:       ккал</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="breakfast_change">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lunch_change">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="dinner_change">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="snack_change">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
